metrics/examples/prometheus/install/alert-manager.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: alertmanager
+  labels:
+    app: strimzi
+spec:
+  replicas: 3
```

Comment:
Out of curiosity, do we need 3 replicas of the alert manager? From where this number comes from?

Reply:
Less is probably ok. I took this value from [there](https://github.com/coreos/prometheus-operator/blob/master/example/user-guides/alerting/alertmanager-example.yaml).